src/tests/unit/tests/issue-filing/services/github/github-issue-filing-service.test.tsx

```diff
@@ -104,7 +110,9 @@ describe('GithubIssueFilingServiceTest', () => {
             onPropertyUpdateCallbackMock
                 .setup(updateCallback => updateCallback(It.isValue(payload)))
                 .verifiable(Times.once());
-            wrapper.find(TextField).props().onChange(null, newRepositoryValue);
+
+            const textField = getMockComponentClassPropsForCall(TextField);
+            textField.onChange(null, newRepositoryValue);
```

Comment:
we can merge into one line 
getMockComponentClassPropsForCall(TextField).onCHange(null,newRepositoryValue)

Reply:
Implemented the suggested changes.